pkgs/applications/editors/vscode/extensions/vadimcn.vscode-lldb/default.nix

```diff
@@ -38,6 +38,13 @@ let
       "--bin=codelldb"
     ];
 
+    postFixup = ''
+      wrapProgram $out/bin/codelldb \
+        --set-default LLDB_DEBUGSERVER_PATH "${lldb.out}/bin/lldb-server"
+      ln -s ${lldb.lib} $out/lldb
```

Comment:
```suggestion
      ln -s ${lldb.lib} $out/lib/lldb
```

Reply:
Doing that won't work either, because it may break it when used with the vscode extension:

```
The argument '--liblldb <liblldb>' was provided more than once, but cannot be used multiple times
```

Reply:
Actually, I don't think the vscode extension calls `codelldb` with the `--liblldb` flag.